buildroot-external/kernel/hassos.config

```diff
@@ -15,6 +15,7 @@ CONFIG_VFAT_FS=y
 CONFIG_SQUASHFS=y
 CONFIG_SQUASHFS_XATTR=y
 CONFIG_SQUASHFS_LZ4=y
+CONFIG_BTRFS_FS=y
```

Comment:
Why not just as a module? On embedded systems, ZFS or BTRFS is quite useless or just over-engineered @agners ?